multik-default/build.gradle.kts

```diff
@@ -1,14 +1,47 @@
-dependencies {
-    api(project(":multik-api"))
-    implementation(project(":multik-jvm"))
-    implementation(project(":multik-native"))
+plugins {
+    kotlin("multiplatform")
```

Comment:
multik-native is for jvm only, why is multik-default multiplatform then?

Reply:
I would have converted `multik-native` too. However it has a bunch of custom tasks and uses the `cpp-library` plugin. So if you wanted me to do it I would have to do some research on the correct way to handle that. I did convert `multik-default` to using the multiplatform plugin, however it only targets jvm, so basically nothing is different for  `multik-default` I just prepped the gradle config to be able to add more targets

Reply:
I just realized I forgot to move multik-default src/main to src/jvmMain.. now that is fixed and it is correctly built